spec/controllers/state_file/questions/income_review_controller_spec.rb

```diff
@@ -100,6 +104,120 @@
         expect(response.body).to have_link(href: edit_w2_path(id: state_file_w2_2.id))
       end
     end
+
+    context "when no W2 box 14 warnings" do
+      let(:intake) { create(:state_file_nj_intake) }
+      
+      shared_examples "does not display W2 warnings" do
+        it "does not display W2 warnings" do
+          get :edit, params: params
+          expect(response.body).not_to have_text "We need to double-check some information"
```

Comment:
[pebble] I think the recommended pattern would be to reference the `I18n.t` rather than the string directly